.gitignore

```diff
@@ -1,2 +1,10 @@
-*.class
-*.log
+dist/*
+target/
+lib_managed/
+src_managed/
```

Comment:
I always copy this from https://github.com/github/gitignore/blob/master/Global/SBT.gitignore